packages/rulesets/src/spectral/test/xms-pageable-for-list-calls.test.ts

```diff
@@ -206,6 +254,86 @@ test("CollectionObjectPropertiesNaming should find no errors", () => {
             },
           },
         },
+      "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Music/Configurations": {
+        get: {
+          operationId: "test_ListByID",
+          responses: {
+            200: {
+              description: "Success",
+              schema: {
+                properties: {
+                  value: {
+                    type: "array",
+                  },
+                  nextLink: {
+                    type: "string",
+                  },
+                },
+                required: ["value"],
+              },
+            },
+          },
+          "x-ms-pageable": {
+            nextLinkName: null,
+          },
+        },
+      },
+      "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Music/Configurations/config": {
```

Comment:
Specs will more commonly have variables for the resource names, not static resource names. I think we have a rule that errors on static resource names
```suggestion
      "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Music/Configurations/{configName}": {
```